umap/templates/umap/content.html

```diff
@@ -68,12 +68,6 @@
           callback: function (data) {
             const container = this.parentNode
             container.innerHTML = data
```

Comment:
At the moment, this is including all the retrieved page inside the current one(I guess it's why it's still a draft :-))

Reply:
All the retrieved page is actually an HTML fragment mainly with the list of web components with correct ids/settings.

Reply:
ah okay, thanks for the clarification :-)